### ISSUE TITLE
Feature request: Disable automatic cursor transfer when adding the last line.

### ISSUE THREAD

**Issue by Vmelnikoff** [editorconfig/editorconfig-vscode#361]
Visual Studio Code v.1.85 + EditorConfig extension.

Settings -     "files.autoSave": "afterDelay",

Current behavior (with autosave setting):
After a second starts autosaving and a new last empty line is added at the beginning of typing, **the cursor is moved**, in the next string because of this, text break glitches, for example:
```css
.red
{/* here - moved cursor from the VS extension EditorConfig */}
/* here - empty new line created by the VS extension EditorConfig */
```

In EditorConfig plugin (WebStorm) another, more correct behavior:
After a second starts autosaving and a new last empty line is added, but , **the cursor is NOT moved**. Therefore, there are no glitches with line breaks.
```css
.red {/* here - NOT moved cursor from the VS extension EditorConfig */}
/* here - empty new line created by the WS plugin EditorConfig */
```

Could you do the same behavior as WebStorm plugin (the cursor is NOT moved when auto-creating a new last empty line) in the plugin EditorConfig for VSCode?

Thank you in advance.

**Comment by SunsetTechuila** [comment]
I don't think that can be fixed on the extension side. At least, without ugly workarounds like `vscode.commands.executeCommand("cursorMove")`

this is where the anchor goes when setting selection to `Position(46, 0)`

![image](https://github.com/editorconfig/editorconfig-vscode/assets/115353812/34b13831-43aa-49b8-95c9-521ee546573b)

and this is when setting selection to `Position(46, 1)`

![image](https://github.com/editorconfig/editorconfig-vscode/assets/115353812/911dec2d-722c-4315-a150-ba4d4baac455)

no way to place it between the brace and the newline

**Comment by SunsetTechuila** [comment]
it seems that vscode does [the same](https://github.com/microsoft/vscode/blob/d40dff9ef9aacb8e5226bcf3938c71f4b9543120/src/vs/workbench/contrib/notebook/browser/contrib/saveParticipants/saveParticipants.ts#L308-L310) what this [extension does](https://github.com/editorconfig/editorconfig-vscode/blob/b86ebd4e9062c76e7145bcefc1a17cf2e6cc1acc/src/DocumentWatcher.ts#L87-L88)
so there is might be an issue with the public api

**Comment by SunsetTechuila** [comment]
p.s.

copying the value (`[...activeEditor.selections]`) instead of the reference [here](https://github.com/editorconfig/editorconfig-vscode/blob/b86ebd4e9062c76e7145bcefc1a17cf2e6cc1acc/src/DocumentWatcher.ts#L78) doesn't fix anything

**Comment by SunsetTechuila** [comment]
duplicate of #330

**Comment by xuhdev** [comment]
Fixed by https://github.com/editorconfig/editorconfig-vscode/pull/372 . Let me know if it's a different issue.
